iothub/service/src/Registry/Models/ExportImportDevice.cs

```diff
@@ -82,7 +82,7 @@ public ExportImportDevice(string deviceId, ImportMode importMode)
         /// The type of registry operation and ETag preferences.
         /// </summary>
         [JsonProperty("importMode")]
-        public ImportMode ImportMode { get; set; }
+        public ImportMode? ImportMode { get; set; }
```

Comment:
Allow this to be null so when deserialized, if no value we don't default to one that isn't true.